trp_project_scrum/security/ir_rules_collaborators_followers.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="project_rule_foll" model="ir.rule">
```

Comment:
foll??? Please no random abbreviations. project_rule_followers should fit in the xmlid... Same for the other xmlid's.